azure-core/azure/core/pipeline/policies/base_async.py

```diff
@@ -49,17 +49,19 @@ class AbstractAsyncContextManager(object):  # type: ignore
 HTTPRequestType = TypeVar("HTTPRequestType")
 
 
+#pylint: disable=too-few-public-methods
 class AsyncHTTPPolicy(abc.ABC, Generic[HTTPRequestType, AsyncHTTPResponseType]):
     """An async HTTP policy ABC.
     """
     def __init__(self) -> None:
         # next will be set once in the pipeline
-        self.next = None  # type: Optional[Union[AsyncHTTPPolicy[HTTPRequestType, AsyncHTTPResponseType], AsyncHttpTransport[HTTPRequestType, AsyncHTTPResponseType]]]
+        # type: Optional[Union[AsyncHTTPPolicy[HTTPRequestType, AsyncHTTPResponseType], \
```

Comment:
Will [mypy](http://mypy-lang.org) still pick up the type annotation if it is on this line?

Reply:
good point, this is not how mypy supports multiline annotations. changing this back.
https://mypy.readthedocs.io/en/latest/python2.html#multi-line-python-2-function-annotations